Source/Moya.swift

```diff
@@ -103,6 +103,28 @@ public class MoyaProvider<Target: TargetType> {
         return endpointClosure(token)
     }
     
+    /// Designated request-making method with queue option. Returns a Cancellable token to cancel the request later.
+    public func requestWithQueue(target: Target, queue:dispatch_queue_t? = nil, completion: Moya.Completion) -> Cancellable {
```

Comment:
I think there is no need to use name `requestWithQueue`, we can just call it  `request`, but with one parameter more.